scripts/hypothesistooling.py

```diff
@@ -57,6 +57,16 @@ def latest_version():
     return latest
 
 
+def hash_for_name(name):
+    return subprocess.check_output([
+        "git", "rev-parse", name
+    ]).decode('ascii').strip()
+
+
+def on_master():
+    return hash_for_name("HEAD") == hash_for_name("origin/master")
```

Comment:
I believe not, because that's the logic for pull requests, while this only runs on actual master where this behaviour is correct.